src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.domain;
+
+public class Car {
+    public String positionView = "";
+    private int criteriaNum = 4;
+
+    public Car() {
+    }
+
+    public boolean isMoveAble(int randomInt) {
+        if (randomInt >= criteriaNum) {
+            return true;
+        }
+        return false;
+    }
+
+    public String moveCar(boolean isMoveable) {
+        if (isMoveable == true) {
+            positionView += "-";
+        }
+        return positionView;
+    }
```

Comment:
```suggestion
    private boolean isMoveAble(int randomInt) {
        return randomInt >= criteriaNum;
    }

    public String moveCar(int randomInt) {
        if (isMoveAble(randomInt)) {
            // move
        }
    }
```
isMoveAble을 public으로 노출할 필요가 있을까요~?